### DISCUSSION TITLE
Update to supabase js v2

### DISCUSSION THREAD

**Discussion by oagna** [tmm/react-supabase#53] [General]
Is there a plan to update react-supabase to [supabase js v2](https://supabase.com/blog/supabase-js-v2) as it released recently.

**Comment by tmm** [comment]
Busy at the moment maintaining another library, but would accept a PR for it.

**Comment by eStasicki** [comment]
@tmm Hi !

Or has something changed in the update process to v2? That would be a super developmental step :)
